cobbler/modules/serializers/file.py

```diff
@@ -86,7 +86,7 @@ def serialize_item(collection, item):
 
     _dict = item.to_dict()
     with open(filename, "w+") as file_descriptor:
-        data = simplejson.dumps(_dict, encoding="utf-8", sort_keys=sort_keys, indent=indent)
```

Comment:
You're removing the encoding due to simplejson defaulting to `utf-8`, right?

Reply:
Nope due to that the keyword simply does not exist: https://docs.python.org/3/library/json.html#json.dumps

Explanation see: https://stackoverflow.com/a/18337754/4730773